dialogflow/stopwatch/functions/index.js

```diff
@@ -0,0 +1,118 @@
+/**
+* Copyright 2019 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
```

Comment:
Likely we'll want to put all the firebase function code in a different location: Not ios-docs-samples.
Probably in node repo.

Reply:
https://github.com/santhoshvaddi/nodejs-docs-samples/pull/1